lexactivator.go

```diff
@@ -1074,19 +1076,26 @@ func CheckForReleaseUpdate(platform string, version string, channel string, call
    * release- returns release struct of the latest available release, depending on the 
      flag LA_RELEASES_ALLOWED or LA_RELEASES_ALL passed to the CheckReleaseUpdate().
 
+   * userData - data that is passed to the callback function when it is registered
+	  using the CheckReleaseUpdate function. This parameter is optional and can be nil if no user data
+	  is passed to the CheckReleaseUpdate function.
+
    PARAMETERS:
```

Comment:
There are no such spaces in the userData description. It seems maybe they appear only here.